Services/Interfaces/web_interface/web_interface.py

```diff
@@ -120,9 +122,9 @@ async def _inner_start(self) -> bool:
         threading.Thread.start(self)
         return True
 
-    def _prepare_stop(self):
-        self.srv.server_close()
-
-    def stop(self):
-        self._prepare_stop()
-        self.srv.shutdown()
+    async def stop(self):
```

Comment:
what's the reason for making it async ?

Reply:
Because the Service api is calling stop with an await
```
async def stop_interfaces(interfaces: list) -> None:
    await manager_stop_interfaces(interfaces)
```